.evergreen/run-connectivity-tests.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Don't trace since the URI contains a password that shouldn't show up in the logs
+set -o errexit  # Exit the script with error if any of the commands fail
+
+# Supported/used environment variables:
+#       JDK                     Set the version of java to be used.  Java versions can be set from the java toolchain /opt/java
+#                               "jdk5", "jdk6", "jdk7", "jdk8"
+# Support arguments:
+#       Pass as many MongoDB URIS as arguments to this script as required
+
+JDK=${JDK:-jdk}
+JAVA_HOME="/opt/java/${JDK}"
+
+############################################
+#            Main Program                  #
+############################################
+
+echo "Running connectivity tests with ${JDK}"
+
+# We always compile with the latest version of java
+export JAVA_HOME="/opt/java/jdk8"
+
+./gradlew -version
+./gradlew --info driver-sync:testClasses
```

Comment:
Is this needed? We don't seem to explicitly assemble the testClasses in the other evergreen files? eg: gssapi-auth-test

Reply:
You're right.  It's useless.  I was trying to make it so the compilation happens once, and in the loop just the test is executed.  But I don't see a way to do that with gradle, as `--rerun-tasks` re-executes all dependent tasks